VideoWeb/VideoWeb/Extensions/ConferenceForVhOfficerResponseExtensions.cs

```diff
@@ -16,11 +17,13 @@ public static IEnumerable<ConferenceForVhOfficerResponse> ApplyCsoFilter(this IE
             }
 
             IEnumerable<ConferenceForVhOfficerResponse> filteredConferences;
-            
+        
             if (!query.AllocatedCsoIds.Any() && query.IncludeUnallocated)
             {
                 filteredConferences = conferences
-                    .Where(r => r.AllocatedCsoId == null);
+                    .Where(r => r.AllocatedCsoId == null &&
+                                r.CaseType != "Generic" &&
```

Comment:
we'll need to apply this to the else statement (line 32) as well, as that's where we add in the unallocated hearings.

should be something like:

`.Union(conferences.Where(r => r.AllocatedCsoId == null && r.CaseType != "Generic" && query.IncludeUnallocated));`